pkg/util/system/system_utils.go

```diff
@@ -18,14 +18,25 @@ package system
 
 import (
 	"strings"
+
+	"k8s.io/api/core/v1"
+	"k8s.io/kubernetes/cmd/kubeadm/app/constants"
```

Comment:
Yeah, that make sense. But in my PR, I just want to import LabelNodeRoleMaster.
I think your issue is that why LabelNodeRoleMaster comes from an application(kubeadm), why not types.
So if the file of LabelNodeRoleMaster is changed, we will change this import, Do you agree with it?